listener/logs/logger.js

```diff
@@ -7,7 +7,10 @@
 const {createLogger, format, transports} = require('winston');
 
 // Choose log level according to environment.
-const LoggerLevel = process.env.NODE_ENV === 'production' ? 'info' : 'debug';
+const defaultLoggerLevel = process.env.NODE_ENV === 'production' ? 'info' : 'debug';
+
+// allow the log level to be specifically set via env variable
+const loggerLevel = process.env.LOG_LEVEL ?? defaultLoggerLevel;
```

Comment:
Could you please add `LOG_LEVEL` to `.env.sample`? It would also be good to list or link to the possible levels, either there or in this file. https://www.npmjs.com/package/winston#logging-levels

Reply:
I didn't put it in there because `DISABLE_LOGGING` wasn't defined either. I put both into the sample file and left them commented out.

Reply:
Thanks, looks great!